common-lib/src/main/scala/com/gu/mediaservice/lib/play/RequestLoggingFilter.scala

```diff
@@ -33,11 +34,13 @@ class RequestLoggingFilter(override val mat: Materializer)(implicit ec: Executio
   private def logSuccess(request: RequestHeader, response: Result, duration: Long): Unit = {
     val originIp = request.headers.get("X-Forwarded-For").getOrElse(request.remoteAddress)
     val referer = request.headers.get("Referer").getOrElse("")
+    val originalService = request.headers.get(Authentication.originalServiceHeaderName).getOrElse("")
     val length = response.header.headers.getOrElse("Content-Length", 0)
 
     val markers = MarkerContext(appendEntries(Map(
       "origin" -> originIp,
       "referrer" -> referer,
+      "originalService" -> originalService,
```

Comment:
Could we only add this marker if there is an `originalService`? That way we can still use `exists` in ELK when searching logs.

Reply:
Yep makes sense, I've updated the code